src/modules/0.0.14/aws_vpc/index.ts

```diff
@@ -359,6 +359,12 @@ export const AwsVpcModule: Module2 = new Module2({
       out.cidrBlock = vpc.CidrBlock;
       out.state = vpc.State as VpcState;
       out.isDefault = vpc.IsDefault ?? false;
+      const tags: { [key: string]: string } = {};
+      (vpc.Tags || []).filter(t => !!t.Key && !!t.Value).forEach(t => {
```

Comment:
Nit. This filter would technically filter out a key or value that is an empty string. Probably fine for the key, but I could see someone legitimately using just the presence of a key in a HashSet-style and not care about the value, and this would drop it.

Perhaps this would be better:

```suggestion
      (vpc.Tags || []).filter(t => t.hasOwnProperty('Key') && t.hasOwnProperty('Value')).forEach(t => {
```